tests/test_protocols/test_generator.py

```diff
@@ -17,16 +17,15 @@
 #
 # ------------------------------------------------------------------------------
 """This module contains the tests for the protocol generator."""
-
-import inspect
+import filecmp
 import logging
```

Comment:
Please break up into a directory with multiple test files; following structure in aea package

Reply:
Done